mscore/pagesettings.ui

```diff
@@ -94,7 +94,7 @@
            </sizepolicy>
           </property>
           <property name="maximum">
-           <double>2000.000000000000000</double>
+           <double>200000.000000000000000</double>
```

Comment:
Jojo - do you remember the discussion we had late last year about rounding errors in Page Settings?  You had suggested limiting the values in the dialog box to 3 digits, since that's what it is elsewhere in MuseScore.  I think it was 3 digits...  Now would be a good time to do that, right?  And it's simply a change in the settings in the Qt dialog editor UI, right?
Here is where a rather lengthy discussion begins: https://musescore.org/en/node/61746#comment-388036